src/SIM/SIMinput.C

```diff
@@ -272,6 +272,17 @@ bool SIMinput::parseGeometryTag (const TiXmlElement* elem)
               }
             }
         }
+
+        item = set->FirstChildElement("point");
+        for (; idim == 0 && item; item = item->NextSiblingElement("point"))
+          if (item->FirstChild())
```

Comment:
if somebody for some reason have a point tag with no value, this will crash.

Reply:
That's not supposed to happen, but put in check on `item->FirstChild()->Value()`anyway